### ISSUE TITLE
Back-off for non-existing synsets in BabelNet?

### ISSUE THREAD

**Issue by cristinae** [albarron/LumpSTS#9]
Now, we look for the synset with more relations for a given lemma with a concrete PoS. If it's a NE, we don't mind the PoS and force the synset to be a NE also.

What happens if the synset we expect (with desired PoS or NE) does not exist? What's the back-off? If the PoS doesn't match do we take another one? And if there is no NE? Should it be an empty result in the same way it is for the non-interesting PoS?

**Comment by cristinae** [comment]
NEs are not considered a special case at this moment, so, no need to think on its back-off.

The system has also changed from selecting the sense with more relations to the sense that BabelNet ranks first (see description in the wiki). Still, we expect the sense to match the PoS, if this doesn't happen, we don't use its ID. Should we use the top1 with any PoS then?

**Comment by albarron** [comment]
issue closed?

**Comment by cristinae** [comment]
No, the question is still open although data is being annotated without the back-off.
